casts/cards/src/types.ts

```diff
@@ -0,0 +1,10 @@
+import type { ImageURISource } from 'react-native';
+
+export type ContentType = 'Image' | 'Video' | 'Unknown';
+
+export type ContentSource = ImageURISource;
+
+export type Content = {
```

Comment:
Too generic type name to export outside, need to either export it with different name with prefix, or rename it here